hapi-proto/src/main/proto/TokenWipeAccount.proto

```diff
@@ -27,8 +27,14 @@ option java_multiple_files = true;
 
 import "BasicTypes.proto";
 
-message TokenWipeAccount {
-    TokenRef token = 1;
-    AccountID account = 2;
-    uint64 amount = 3;
+/* Wipes the provided amount of tokens from the specified Account. Must be signed by the Token's Wipe key.
+ If Wipe Key is not present in the Token, transaction results in TOKEN_HAS_NO_WIPE_KEY.
+ If the provided account is the Token's Treasury Account, transaction results in CANNOT_WIPE_TOKEN_TREASURY_ACCOUNT
+ If the provided account does not Token relation, transaction results in ACCOUNT_HAS_NO_TOKEN_RELATIONSHIP.
```

Comment:
`If the provided account does not Token relation` -> `If the provided account does not have a relation with the provided Token`